core/common/lib/policy-engine-lib/src/main/java/org/eclipse/edc/policy/engine/PolicyEngineImpl.java

```diff
@@ -148,6 +154,29 @@ public Result<Void> validate(Policy policy) {
         return validatorBuilder.build().validate(policy);
     }
 
+    @Override
+    public PolicyEvaluationPlan evaluationPlan(String scope, Policy policy) {
```

Comment:
This is a bit of a nit: does the method create a plan or evaluate it?

Reply:
Just create it never evaluates

Reply:
better to rename it to `createEvaluationPlan` ?